### PR TITLE
Update package name to avoid conflict with other

### PR DESCRIPTION
**Pull request by nowarzz**
Updated package name:
`com.reactlibrary` to `com.reactlibrary.stmpmailer`

Updated READ ME in manual section

### CODE REVIEW

File: android/.project [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+			<name>org.eclipse.buildship.core.gradleprojectbuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+	</buildSpec>
+	<natures>
+		<nature>org.eclipse.jdt.core.javanature</nature>
+		<nature>org.eclipse.buildship.core.gradleprojectnature</nature>
+	</natures>
+</projectDescription>
```
Comment (angelos3lex) [inline android/.project]: Why is this file needed? .project

File: android/.settings/org.eclipse.buildship.core.prefs [hunk 2]
```diff
@@ -0,0 +1,2 @@
+connection.project.dir=
+eclipse.preferences.version=1
```
Comment (angelos3lex) [inline android/.settings/org.eclipse.buildship.core.prefs]: Why .settings/org.eclipse.buildship.core.prefs is needed?

File: example/android/.project [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+		<buildCommand>
+			<name>org.eclipse.buildship.core.gradleprojectbuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+	</buildSpec>
+	<natures>
+		<nature>org.eclipse.buildship.core.gradleprojectnature</nature>
+	</natures>
+</projectDescription>
```
Comment (angelos3lex) [inline example/android/.project]: Why is .project file needed?

File: android/.project (hunk 1 above)
Comment (nowarzz) [reply android/.project]: I am sorry. Forgot to add it to .gitignore. It is configuration file of my IDE. Just ignore it

File: android/.settings/org.eclipse.buildship.core.prefs (hunk 2 above)
Comment (nowarzz) [reply android/.settings/org.eclipse.buildship.core.prefs]: I am sorry. Forgot to add it to .gitignore. It is configuration file of my IDE. Just ignore it

File: example/android/.project (hunk 3 above)
Comment (nowarzz) [reply example/android/.project]: I am sorry. Forgot to add it to .gitignore. It is configuration file of my IDE. Just ignore it

**Comment by angelos3lex** [comment]
This is icluded in v1.2.4 by PR #41
